src/Soil-File/WinOverlapped.class.st

```diff
@@ -0,0 +1,103 @@
+Class {
+	#name : 'WinOverlapped',
```

Comment:
Same for the prefix. Is WinOverlapped a term in that area. If not we might find better intention revealing name

Reply:
`lpOverlapped` is the name in the Windows API. It was originally badly named.  `SoilWinOverlappedStruct` would probably be better.